controllers/storageclassclaim/storageclassclaim_controller.go

```diff
@@ -92,20 +124,296 @@ func (r *StorageClassClaimReconciler) Reconcile(ctx context.Context, request rec
 	return result, nil
 }
 
-func (r *StorageClassClaimReconciler) reconcileConsumerPhases(ctx context.Context, storagecluster *v1.StorageCluster) (reconcile.Result, error) {
-	log := ctrllog.FromContext(ctx)
-	log.Info("Running StorageClassClaim controller in Consumer Mode")
+func (r *StorageClassClaimReconciler) reconcileConsumerPhases() (reconcile.Result, error) {
+	r.log.Info("Running StorageClassClaim controller in Consumer Mode")
+
+	providerClient, err := providerclient.NewProviderClient(
+		r.ctx,
+		r.storageCluster.Spec.ExternalStorage.StorageProviderEndpoint,
+		10*time.Second,
```

Comment:
10 seconds seems problematic if the server is serving many requests, keep it 1 or 2 minutes?

Reply:
PTAL https://github.com/red-hat-storage/ocs-operator/pull/1608#discussion_r837605604

Taken from here: https://github.com/red-hat-storage/ocs-operator/blob/main/controllers/storagecluster/external_ocs.go#L47

Reply:
@Madhu-1 this is a tested value that never created issues for us. A bigger timeout will just keep a reconcile iteration hanging for 1 to 2 minutes. 
Please remember that this is a management endpoint (for 2nd day operations) not a data endpoint